src/components/Pagination/Pagination.js

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import CssBaseline from '@material-ui/core/CssBaseline';
+import { createMuiTheme, MuiThemeProvider } from '@material-ui/core/styles';
+import PaginationComp from 'material-ui-flat-pagination';
+import InputLabel from '@material-ui/core/InputLabel';
+import MenuItem from '@material-ui/core/MenuItem';
+import FormControl from '@material-ui/core/FormControl';
+import Select from '@material-ui/core/Select';
+import Grid from '@material-ui/core/Grid';
+import useStyles from './style';
+import { useStateValue } from '../StateProvider';
```

Comment:
You should use named imports for material-UI. It serves as refactoring but also as bundle-size optimization. Please read [this](https://material-ui.com/guides/minimizing-bundle-size/).